modules/oadp-upgrade-from-oadp-data-mover-1-2-0.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/oadp-release-notes.adoc
+
+:_mod-docs-content-type: PROCEDURE
+
+[id="upgrade-from-oadp-data-mover-1-2-0_{context}"]
+= Upgrading from OADP 1.2 Technology Preview Data Mover
+
+{oadp-first} 1.2 Data Mover backups *cannot* be restored with OADP 1.3. To prevent a gap in the data protection of your applications, complete the following steps before upgrading to the OADP 1.3 upgrade:
+
+. If your cluster backups are sufficient and Container Storage Interface (CSI) storage is available,
+back up the applications with a CSI backup.
+. If you require off cluster backups:
+.. Back up the applications with a file system backup that uses the `--default-volumes-to-fs-backup=true or backup.spec.defaultVolumesToFsBackup` options.
+.. Back up the applications with your object storage plugins, for example, `velero-plugin-for-aws`.
+
+[NOTE]
+====
+If your OADP 1.2 Data Mover backup must be restored, OADP must be fully uninstalled and OADP 1.2 reinstalled and configured.
```

Comment:
```suggestion
To restore OADP 1.2 Data Mover backup, you must uninstall OADP, and install and configure OADP 1.2.
```

Reply:
Shuba's construction is better, and it needs to be rewritten to follow that pattern and be technically correct.

Reply:
Done